### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by WithoutPants** [stashapp/stash#2177] [Announcements]
# Release Notes
### ✨ New Features
* Changed query string parsing behaviour to require all words by default, with the option to `or` keywords and exclude keywords. See the `Browsing` section of the manual for details. ([#1982](https://github.com/stashapp/stash/pull/1982))
* Show heatmaps and median stroke speed for interactive scenes on the scenes page. ([#2096](https://github.com/stashapp/stash/pull/2096))
* Added selective clean task. ([#2125](https://github.com/stashapp/stash/pull/2125))
* Added option to force generation of transcodes for selected scenes. ([#2126](https://github.com/stashapp/stash/pull/2126))
* Save task options when scanning, generating and auto-tagging. ([#1949](https://github.com/stashapp/stash/pull/1949), [#2061](https://github.com/stashapp/stash/pull/2061))
* Added forward jump 10 second button to video player. ([#1973](https://github.com/stashapp/stash/pull/1973))

### 🎨 Improvements
* Overhauled, restructured and added auto-save to the settings pages. ([#2086](https://github.com/stashapp/stash/pull/2086))
* Added keyboard shortcuts to hide scene page sidebar and scene scrubber. ([#2099](https://github.com/stashapp/stash/pull/2099))
* Added support for setting scrapers path in the settings page. ([#2124](https://github.com/stashapp/stash/pull/2124))
* Made scene phash field in File Info tab a link to show duplicate scenes. ([#2154](https://github.com/stashapp/stash/pull/2154))
* Rollback operation if files fail to be deleted. ([#1954](https://github.com/stashapp/stash/pull/1954))
* Prefer right-most Studio match in the file path when autotagging. ([#2057](https://github.com/stashapp/stash/pull/2057))
* Show Created/Updated dates in scene/image/gallery details pages. ([#2145](https://github.com/stashapp/stash/pull/2145))
* Include path and hashes in destroy scene/image/gallery post hook input. ([#2102](https://github.com/stashapp/stash/pull/2102/files))
* Added plugin hook for Tag merge operation. ([#2010](https://github.com/stashapp/stash/pull/2010))

### 🐛 Bug fixes
* Don't include audio in marker previews if Include Audio option is unchecked. ([#2101](https://github.com/stashapp/stash/pull/2101))
* Include performer aliases when scraping from stash-box. ([#2091](https://github.com/stashapp/stash/pull/2091/files))
* Remove empty folder-based galleries during clean. ([#1954](https://github.com/stashapp/stash/pull/1954))
* Select first scene result in scene tagger where possible. ([#2051](https://github.com/stashapp/stash/pull/2051))
* Reject dates with invalid format. ([#2052](https://github.com/stashapp/stash/pull/2052))
* Fix Autostart Video on Play Selected and Continue Playlist default settings not working. ([#2050](https://github.com/stashapp/stash/pull/2050))
* Fix "Custom Performer Images" feature picking up non-image files. ([#2017](https://github.com/stashapp/stash/pull/2017))

<hr /><em>This discussion was created from the release <a href='https://github.com/stashapp/stash/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by UncleRoger33** [comment]
Great work 👍  Especially new "Settings" looks neat and less confusing now. Thank you all.

**Comment by philpw99** [comment]
Love the new settings !
Love the new multi-language interface ! 
It's perfect !

<img src="https://media3.giphy.com/media/26gsvAm8UPaczzXz2/giphy.webp?cid=ecf05e47yjpj05irl14o4rfbsw7xetilourks4woy50ldf9p&rid=giphy.webp&ct=g" />
